boom! testing postgres_range-0.8.2 against nightly for nightly
boom! running: cargo +nightly build --frozen
boom! creating container for: cargo +nightly build --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly build --frozen cargobomb`
blam! 4d3f1948e936316698053531238adf539dbfad2d44d5165fb677f5cd9186c713
boom! running `docker start -a 4d3f1948e936316698053531238adf539dbfad2d44d5165fb677f5cd9186c713`
kablam!    Compiling postgres v0.11.11
kablam!    Compiling postgres_range v0.8.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 17.37 secs
boom! running `docker rm -f 4d3f1948e936316698053531238adf539dbfad2d44d5165fb677f5cd9186c713`
blam! 4d3f1948e936316698053531238adf539dbfad2d44d5165fb677f5cd9186c713
boom! running: cargo +nightly test --frozen --no-run
boom! creating container for: cargo +nightly test --frozen --no-run
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen --no-run cargobomb`
blam! 6e316fabc5295425233123d148c250339acd4d8016b1c345abe9b8a367601b04
boom! running `docker start -a 6e316fabc5295425233123d148c250339acd4d8016b1c345abe9b8a367601b04`
kablam!    Compiling postgres_range v0.8.2 (file:///source)
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 6.17 secs
boom! running `docker rm -f 6e316fabc5295425233123d148c250339acd4d8016b1c345abe9b8a367601b04`
blam! 6e316fabc5295425233123d148c250339acd4d8016b1c345abe9b8a367601b04
boom! running: cargo +nightly test --frozen
boom! creating container for: cargo +nightly test --frozen
boom! running `docker create -v /home/ec2-user/cargobomb/./work/local/test-source/nightly/nightly:/source:ro -v /home/ec2-user/cargobomb/./work/local/cargo-home:/cargo-home:ro -v /home/ec2-user/cargobomb/./work/local/rustup-home:/rustup-home:ro -v /home/ec2-user/cargobomb/./work/local/target-dirs/nightly/nightly:/target:rw -e USER_ID=500 -e CMD=cargo +nightly test --frozen cargobomb`
blam! e7d6ad813d1ac16290a0c1dc79da712af06f62d3490d490f8c731b609b967a69
boom! running `docker start -a e7d6ad813d1ac16290a0c1dc79da712af06f62d3490d490f8c731b609b967a69`
kablam!     Finished dev [unoptimized + debuginfo] target(s) in 0.0 secs
kablam!      Running /target/debug/deps/postgres_range-3b6cc83256fcf689
blam! 
blam! running 16 tests
blam! test impls::test::test_int4range_params ... FAILED
blam! test impls::test::test_int8range_params ... FAILED
blam! test impls::test::test_tsrange_params ... FAILED
blam! test impls::test::test_tstzrange_params ... FAILED
blam! test test::test_normalize_lower ... ok
blam! test test::test_intersection ... ok
blam! test test::test_contains_range ... ok
blam! test test::test_normalize_upper ... ok
blam! test test::test_range_bound_lower_lt ... ok
blam! test test::test_range_bound_upper_in_bounds ... ok
blam! test test::test_range_bound_lower_in_bounds ... ok
blam! test test::test_range_bound_upper_lt ... ok
blam! test test::test_union ... ok
blam! test test::test_range_empty ... ok
blam! test test::test_range_contains ... ok
blam! test test::test_range_normalizes ... ok
blam! 
blam! failures:
blam! 
blam! ---- impls::test::test_int4range_params stdout ----
blam! 	thread 'impls::test::test_int4range_params' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! note: Run with `RUST_BACKTRACE=1` for a backtrace.
blam! 
blam! ---- impls::test::test_int8range_params stdout ----
blam! 	thread 'impls::test::test_int8range_params' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- impls::test::test_tsrange_params stdout ----
blam! 	thread 'impls::test::test_tsrange_params' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! ---- impls::test::test_tstzrange_params stdout ----
blam! 	thread 'impls::test::test_tstzrange_params' panicked at 'called `Result::unwrap()` on an `Err` value: Io(Error { repr: Os { code: 111, message: "Connection refused" } })', /checkout/src/libcore/result.rs:859
blam! 
blam! 
blam! failures:
blam!     impls::test::test_int4range_params
blam!     impls::test::test_int8range_params
blam!     impls::test::test_tsrange_params
blam!     impls::test::test_tstzrange_params
blam! 
blam! test result: FAILED. 12 passed; 4 failed; 0 ignored; 0 measured
blam! 
kablam! error: test failed
boom! running `docker rm -f e7d6ad813d1ac16290a0c1dc79da712af06f62d3490d490f8c731b609b967a69`
blam! e7d6ad813d1ac16290a0c1dc79da712af06f62d3490d490f8c731b609b967a69
